tools/shoestring/README.md

```diff
@@ -86,14 +86,15 @@ min-cosignatures-count --config CONFIG --ca-key-path CA_KEY_PATH [--update]
 
 ### import-bootstrap
 
-Imports settings from a symbol-bootstap installation.
+Imports settings from a symbol-bootstrap installation.
 
 
 ```
 import-bootstrap --config CONFIG --bootstrap BOOTSTRAP
 
   --config CONFIG       path to shoestring configuration file
   --bootstrap BOOTSTRAP path to bootstrap target directory
+  --node-key            include the node key
```

Comment:
Since the command already states importing from bootstrap, I did not think adding the ``import-node-key`` is required.   I will update to `` --include-node-key``.